### PR TITLE
removed subject restriction on the permission check used by the searc…

### PR DESCRIPTION
**Pull request by jtreuting**
…h students SQL

Still have not tested locally but check it out

### CODE REVIEW

File: reporting-service/src/main/resources/application.sql.yml [hunk 1]
```diff
@@ -335,3 +335,43 @@ sql:
            OR 1 = :state_embargo_admin
            OR s.district_id IN (:district_embargo_admin_ids)
         )
+
+      studentSearchPermissions: >-
```
Comment (jtreuting) [inline reporting-service/src/main/resources/application.sql.yml]: combined the `sql.snipper.studentExamPermissionWIthoutEmbargo` and the check above, but removed the restriction on subject that was part of the studentExamPermissionWIthoutEmbargo check

File: reporting-service/src/main/resources/application.sql.yml [hunk 2]
```diff
@@ -335,3 +335,43 @@ sql:
...
+            )
+          )
+          OR
+          # Group permissions
+          (
+            EXISTS(
+               SELECT 1 FROM student_group_membership gm
+                     JOIN student_group sg ON sg.id = gm.student_group_id
+                     JOIN user_student_group usg ON usg.student_group_id = gm.student_group_id
+                        WHERE gm.student_id = e.student_id AND usg.user_login = :user_login
```
Comment (jtreuting) [inline reporting-service/src/main/resources/application.sql.yml]: here is where there was also a check on the subject

Comment (agorina) [reply reporting-service/src/main/resources/application.sql.yml]: 👍

**Comment by agorina** [comment]
LGTM. I am assuming it passed the ITs?

**Pull request merged**